src/test/data/base58_encode_decode.json

```diff
@@ -11,6 +11,13 @@
 ["ecac89cad93923c02321", "EJDM8drfXA6uyA"],
 ["10c8511e", "Rt5zm"],
 ["00000000000000000000", "1111111111"],
+["00000000000000000000000000000000000000000000000000000000000000000000000000000000", "1111111111111111111111111111111111111111"],
+["00000000000000000000000000000000000000000000000000000000000000000000000000000001", "1111111111111111111111111111111111111112"],
+["0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ec39d04c37e71e5d591881f6", "111111111111111111111111111111111111111111111111111111111111111111111111111111111111115TYzLYH1udmLdzCLM"],
```

Comment:
I've added the random test cases that failed during development to make sure they're covered deterministically.
Also added ones which are testing the boundaries of powers of 58, since those cases happen less frequently in the randomized tests.

Reply:
Since some of these failed during development of this feature, I think it adds and extra layer of safety to see that they're passing after the change as well.
But if you think it's better to merge it before this change, I can split it out of course.